subworkflows/nf-core/fastq_fastqc_umitools_trimgalore/tests/main.nf.test.snap

```diff
@@ -377,14 +377,14 @@
                 
             ],
             [
-                
+                null
```

Comment:
Why has this changed?

Reply:
No idea, I did not investigate, just fix this upstream

Reply:
cf https://github.com/nf-core/modules/pull/6761